multigeiger/multigeiger.ino

```diff
@@ -804,25 +841,32 @@ void jbTone(unsigned int frequency_mHz, unsigned int time_ms, unsigned char volu
 // ===================================================================================================================================
 // Send to Server Subfunctions
 
-String buildhttpHeaderandBodySBM(HTTPClient *head, int radiation_cpm, boolean addname) {
+String buildhttpHeaderandBodySBM(HTTPClient *head, unsigned int hvpulse, boolean addname, bool debug) {
   head->addHeader("Content-Type", "application/json; charset=UTF-8");
   head->addHeader("X-PIN","19");
   String chipID = String(ssid);
   chipID.replace("ESP32","esp32");
   head->addHeader("X-Sensor",chipID);
   head->addHeader("Connection","close");
-  String valuetype = (addname ? tube.substring(10)+"_" : "");
+  String tubetype = tubes[TUBE_TYPE].type;
+  tubetype = tubetype.substring(10);
+  String valuetype = (addname ? tubetype+"_" : "");
   valuetype += "counts_per_minute";
-  String body = "{\"software_version\":\""+String(revString)+"\",\
-  \"sensordatavalues\":[{\"value_type\":\""+valuetype+"\",\
-  \"value\":\""+String(radiation_cpm)+"\"}]}";                       //Build the actual POST request
+  String body = "{\"sensordatavalues\":[";
+  body += "{\"value_type\":\""+valuetype+"\",\"value\":\""+current_cpm+"\"}";
+  if (debug) {
+    body += ",{\"value_type\":\"hv_pulses\",\"value\":\""+String(hvpulse)+"\"},";
```

Comment:
hmm, why is that interesting?

isn't that rather an implementation detail of this GM counter than a value worth publishing?

Reply:
> rexfue notes that juergen wants to know the hv pulses to see potential humidity or other issues related with hv generation/usage.